.github/workflows/grpc-services-pypi-publish.yml

```diff
@@ -1,51 +1,52 @@
-name: Build and publish Python distributions to PyPI
+name: Build and publish Python distributions to PyPI. New gRPC generator
 
 on:
   push:
     branches:
-    - master
-    - version-*
+      - master
+      - version-*
```

Comment:
shouldn't dev-version branch also be here?

Reply:
No, because we should add a separate workflow for that